Makefile

```diff
@@ -72,10 +73,11 @@ supportedsites:
 README.txt: README.md
 	pandoc -f markdown -t plain README.md -o README.txt
 
-youtube-dl.1: README.md
-	python devscripts/prepare_manpage.py >youtube-dl.1.temp.md
+youtube-dl.1: youtube_dl/*.py youtube_dl/*/*.py
```

Comment:
Yes, I think so. But since the target **README.md** is made before **youtube-dl.1**, the problem is being shadowed. Consider the following shell session:
```
alexvong1995@debian:~/tmpd/youtube-dl$ make --debug 
GNU Make 4.0
Built for x86_64-pc-linux-gnu
Copyright (C) 1988-2013 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Reading makefiles...
Updating goal targets....
 File 'all' does not exist.
   File 'youtube-dl' does not exist.
  Must remake target 'youtube-dl'.
zip --quiet youtube-dl youtube_dl/*.py youtube_dl/*/*.py
zip --quiet --junk-paths youtube-dl youtube_dl/__main__.py
echo '#!/usr/bin/env python' > youtube-dl
cat youtube-dl.zip >> youtube-dl
rm youtube-dl.zip
chmod a+x youtube-dl
  Successfully remade target file 'youtube-dl'.
   Prerequisite 'youtube_dl/utils.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/version.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/compat.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/postprocessor/ffmpeg.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/youtube.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/mtv.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/xhamster.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/nfl.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/generic.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/qqmusic.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/bbc.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/vidme.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/common.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/odnoklassniki.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/nhl.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/kuwo.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/playwire.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/viewster.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/shahid.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/iconosquare.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/clubic.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/keek.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/pornhub.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/condenast.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/fktv.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/ninegag.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/__init__.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/mitele.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/soundcloud.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/arte.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/eagleplatform.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/telecinco.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/nowness.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/rai.py' is newer than target 'README.md'.
   Prerequisite 'youtube_dl/extractor/iqiyi.py' is newer than target 'README.md'.
  Must remake target 'README.md'.
COLUMNS=80 python youtube_dl/__main__.py --help | python devscripts/make_readme.py README.md > README.new.md
mv -f README.new.md README.md
  Successfully remade target file 'README.md'.
   Prerequisite 'README.md' is newer than target 'CONTRIBUTING.md'.
  Must remake target 'CONTRIBUTING.md'.
python devscripts/make_contributing.py README.md CONTRIBUTING.md
  Successfully remade target file 'CONTRIBUTING.md'.
   File 'README.txt' does not exist.
  Must remake target 'README.txt'.
pandoc -f markdown -t plain README.md -o README.txt
  Successfully remade target file 'README.txt'.
   File 'youtube-dl.1' does not exist.
  Must remake target 'youtube-dl.1'.
COLUMNS=65536 python youtube_dl/__main__.py --help | python devscripts/make_readme.py README.md > README-wide.md
python devscripts/prepare_manpage.py README-wide.md > youtube-dl.1.temp.md
pandoc -s -f markdown -t man youtube-dl.1.temp.md -o youtube-dl.1
rm -f youtube-dl.1.temp.md README-wide.md
  Successfully remade target file 'youtube-dl.1'.
   File 'youtube-dl.bash-completion' does not exist.
  Must remake target 'youtube-dl.bash-completion'.
python devscripts/bash-completion.py
  Successfully remade target file 'youtube-dl.bash-completion'.
   File 'youtube-dl.zsh' does not exist.
  Must remake target 'youtube-dl.zsh'.
python devscripts/zsh-completion.py
  Successfully remade target file 'youtube-dl.zsh'.
   File 'youtube-dl.fish' does not exist.
  Must remake target 'youtube-dl.fish'.
python devscripts/fish-completion.py
  Successfully remade target file 'youtube-dl.fish'.
   File 'supportedsites' does not exist.
  Must remake target 'supportedsites'.
python devscripts/make_supportedsites.py docs/supportedsites.md
  Successfully remade target file 'supportedsites'.
Must remake target 'all'.
Successfully remade target file 'all'.
```